tests/while_loop_break.sol

```diff
@@ -0,0 +1,33 @@
+
+contract Test {
+    function whileTest(int a, int b) public returns (int) {
+        int result = 0;
+        int index = 0;
+
+        while(index <= a){
+            index = index + 1;
+            result = result + index;
+
+            if(result > b){
+                break;
+            }
+        }
+
+        return result;
+    }
+}
+
+
+{
+
+"Test1" : {
+    "exec" : {
+        "code" : #solidity( int sum = functionCall(new Test(); whileTest; 20, 15);)
+    },
+    "post" : {
+        "mem" :  #exists("sum", 21)
+    }
+}
```

Comment:
@ehildenb @shangweilin It supports multiple test cases.

Reply:
@shangweilin (or @KanShuanglong) let's add some multiple tests to this PR of while-loops (so that we're testing each program more than once), to make sure we don't break this functionality accidentally later.

Reply:
I have finished a batch. I will finish all soon.